pkg/scheduler/framework/plugins/volumebinding/binder_test.go

```diff
@@ -2318,3 +2330,86 @@ func TestCapacity(t *testing.T) {
 		})
 	}
 }
+
+func TestGetEligibleNodes(t *testing.T) {
+	type scenarioType struct {
+		// Inputs
+		pvcs  []*v1.PersistentVolumeClaim
+		pvs   []*v1.PersistentVolume
+		nodes []*v1.Node
+
+		// Expected return values
+		eligibleNodes sets.String
+	}
+
+	scenarios := map[string]scenarioType{
```

Comment:
Can you add a scenario where there is a local pv and non-local pv?

Reply:
Done, added test scenario, thanks.